### PR TITLE
feat: add GetRawData func.

### PR DESCRIPTION
**Pull request by appleboy**
fix https://github.com/gin-gonic/gin/issues/854
fix https://github.com/gin-gonic/gin/issues/438
fix https://github.com/gin-gonic/gin/issues/785

### CODE REVIEW

File: context.go [hunk 1]
```diff
@@ -445,6 +446,12 @@ func (c *Context) GetHeader(key string) string {
 	return c.requestHeader(key)
 }
 
+// GetRawData return stream data
+func (c *Context) GetRawData() (body []byte, err error) {
+	body, err = ioutil.ReadAll(c.Request.Body)
```
Comment (tboerger) [inline context.go]: Just directly return?

Comment (appleboy) [inline context.go]: Done. https://github.com/gin-gonic/gin/pull/857/commits/c6ae08aa58ef52ff7108b95e185b291ea764b647

**Comment by praveenk007** [comment]
@appleboy @tboerger How is this exactly streaming if we are storing the data in byte array **in memory** ?

**Comment by shkarface** [comment]
> @appleboy @tboerger How is this exactly streaming if we are storing the data in byte array **in memory** ?

Exactly my thought

**Pull request merged**